apps/andi/test/unit/andi/input_schemas/ingestion_test.exs

```diff
@@ -39,12 +49,13 @@ defmodule Andi.InputSchemas.Ingestion.IngestionTest do
     assert %{name: "Updated Name"} == changeset.changes
   end
 
-  describe "changeset" do
+  describe "validate" do
```

Comment:
This whole test file was making validations, which in now handled by Ingestion.validate instead